### DISCUSSION TITLE
代理网络下无法连网

### DISCUSSION THREAD

**Discussion by GIT-Y-00** [Richasy/Bili.Uwp#70] [问答] [answered]
全程按照这个步骤https://github.com/Richasy/Bili.Uwp/blob/main/how_to_install.md一步步做完的，但是确实无法联网
检查下来是因为clash（网络代理）的原因，哪怕开代理开直连，都稳定连接不上服务器
请问这个应该怎么办？

**Comment by Groupguanfang** [comment]
fidder试过了没有

**Comment by KebabShellGitHub** [comment, marked as answer]
用 Fiddler 里面的 WinConfig

**Reply by GIT-Y-00** [reply]
感谢！

**Comment by TigerCubDen** [comment]
更轻量化的解决方案：https://github.com/Richasy/LoopbackManager

**Comment by luoying1662** [comment]
clash里面常规选择uwp网络回环,进去找到哔哩勾选,保存更改就行了,规则下能正常用

**Reply by Ladulaking** [reply]
nice 兄弟

**Comment by DeepslateQAQ** [comment]
clash修改设置把网络回环开启，如果懒得动的话全选就行

**Comment by Ladulaking** [comment]
收到，已改



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: 团子 ***@***.***&gt;
发送时间: 2023年7月7日 20:02
收件人: Richasy/Bili.Uwp ***@***.***&gt;
抄送: Ladulaking ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [Richasy/Bili.Uwp] 代理网络下无法连网 (Discussion #70)





 
clash修改设置把网络回环开启，如果懒得动的话全选就行
